libbeat/processors/add_docker_metadata/_meta/fields.yml

```diff
@@ -1,7 +1,7 @@
 - key: docker
```

Comment:
I would assume if we change this from `- key: docker` to `key: docker-processor` it could be that we will end up with 2 titles.

Reply:
The `title` field needed to be changed. I tried that before, but I made the change in the yml for the module rather than the processor, and it affected messages as well as the topic title. But you're right...it works if you change the title in the processor instead. I think this is a good enough fix for now, but we'll come up with a better approach later that's more consistent.